Userland/Services/WebContent/PageClient.cpp

```diff
@@ -52,32 +52,24 @@ PageClient::PageClient(PageHost& owner, u64 id)
     , m_backing_store_manager(*this)
 {
     setup_palette();
-}
 
-PageClient::~PageClient() = default;
+    m_paint_refresh_timer = Core::Timer::create_repeating(1000 / 60, [] {
```

Comment:
This is doing an integer division. If you want actually 16.6666666667 ms, one of these values needs to be explicitly a floating point value.

Reply:
yeah, but create_repeating() accepts int as an argument so that should be fine.